src/liballoc/str.rs

```diff
@@ -247,6 +247,7 @@ impl str {
     /// assert_eq!(s, s.replacen("cookie monster", "little lamb", 10));
     /// ```
     #[stable(feature = "str_replacen", since = "1.16.0")]
+    #[must_use]
```

Comment:
Just for consistency, the `#[must_use]` attribute usually comes before the stability attribute.

Reply:
Ok, will update. I'll wait with that + the replace() must_use attr until travis is done, in case this introduces bootstrap compiler warnings.